db/migrate/20170428000001_change_blocks_and_mutes.rb

```diff
@@ -0,0 +1,18 @@
+class ChangeBlocksAndMutes < ActiveRecord::Migration[5.0]
+  def up
+    add_column       :blocks, :block, :boolean
+    execute          "ALTER TABLE blocks ADD CONSTRAINT check_mutes_on_block CHECK(block = TRUE), INHERIT block_mutes"
+    Block.update_all block: true
+    change_column    :blocks, :block, :boolean, null: false
```

Comment:
```
PG::DatatypeMismatch: ERROR:  column "block" in child table must be marked NOT NULL
: ALTER TABLE blocks ADD CONSTRAINT check_mutes_on_block CHECK(block = TRUE), INHERIT block_mutes
```

Probably, you should do with below order:

1. add_column
2. Block.update_all
3. change_column
4. execute